.github/workflows/ci-ui.yml

```diff
@@ -0,0 +1,17 @@
+name: ci-ui
```

Comment:
could you add in `alfred ci.ui` instead of using github action for that ? it will run in local : https://github.com/streamsync-cloud/streamsync/blob/dev/alfred/ci.py and in the ci pipeline also.

alfred documentation : https://alfred-cli.readthedocs.io/en/latest/command.html